src/context.rs

```diff
@@ -652,7 +653,7 @@ pub struct Repo {
 
     /// Contains `true` if the value of `core.fsmonitor` is set to `true`.
     /// If not `true`, `fsmonitor` is explicitly disabled in git commands.
-    fs_monitor_value_is_true: bool,
+    pub(crate) fs_monitor_value_is_true: bool,
```

Comment:
This was a quick-fix to allow using the field, I hope it's alright.

Reply:
It's completely fine.